Points.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+#include <vector>
+#include <iostream>
+#include <math.h>
+
+using namespace std;
+
+
+
+class Point
+{
+	double x, y;
+public:
+	Point()
+	{
+		x = 0;
+		y = 0;
+	}
+	Point(double OX, double  OY)
+	{
+		x = OX;
+		this->y = OY;
+	}
+	virtual ~Point() {}
+
+	double GetX();
+	double GetY();
+	void GetCoord();
+	friend double Dist(Point*, Point*);
```

Comment:
Please don't shortage your names, Not always can understand meaning of them

Reply:
Fix.